.renovaterc.json5

```diff
@@ -7,8 +7,7 @@
   ],
   "timezone": "Europe/Paris",
   "ignorePaths": [
-    "**/node_modules/**",
-    "**/templates/**"
```

Comment:
Maybe cleaner to explicitly add your exception that allowing the whole `templates/` folders?

Reply:
In most cases, the dependencies that we put in the package.json inside the templates are not hard-coded but evaluated at generation time (to match the deps defined in the package.json of the generating package)
However, I believe it wouldn't hurt to scan the dependencies even in `**/templates/**` folders, although files with `.template` suffix won't be scanned

Reply:
fare enough, anyway if we got issue with Renovate, we will rollback this change.